src/sentry/static/sentry/app/types/hooks.ts

```diff
@@ -165,7 +165,7 @@ type OrganizationHeaderComponentHook = (org: Organization) => React.ReactNode;
 /**
  * A FeatureDisabledHook returns a react element when a feature is not enabled.
  */
-type FeatureDisabledHook = (opts: {
+export type FeatureDisabledHook = (opts: {
```

Comment:
Hook implementation in getsentry can use this type for their 'entry point'. They need to use `Parameters<FeatureDisabledHook>[0]` but it works better than before.

Reply:
Looks like this is the only type like this that we export. Does it make more sense for it to just use `FeatureDisabledHooks['<name of hook>']`